src/domain/guest/components/QuestionHeader/QuestionHeader.js

```diff
@@ -1,18 +1,25 @@
-import { useQuery } from 'hooks/axios.hooks';
+import { format } from 'date-fns';
+import React from 'react';
+import Wrapper from './QuestionHeader.styles';
 
-const QuestionHeader = ({ user_id }) => {
-  const { data } = useQuery({ url: `/guests/${user_id}` });
+const QuestionHeader = ({ information }) => {
   return (
-    <>
-      <p> Họ và tên: {!!data && data.fullname} </p>
-      <p> Năm sinh: {!!data && data.date_of_birth} </p>
-      <p> Giới tính: {!!data && data.gender} </p>
-      <p> Dân tộc: {!!data && data.nation} </p>
-      <p> Nhập ngũ: {!!data && data.join_army} </p>
-      <p> Đơn vị: {!!data && data.unit} </p>
-      <p> Cấp bậc: {!!data && data.rank} </p>
-      <p> Chức vụ: {!!data && data.position} </p>
-    </>
+    <Wrapper>
+      <p> Họ và tên: {(!!information && information.name) || ''} </p>
+      <div className="row-infor">
+        <p> Năm sinh: {(!!information && format(information?.yearOfBirth, 'dd/MM/yyyy')) || ''} </p>
+        <p> Giới tính: {(!!information && information.gender) || ''} </p>
+      </div>
+      <div className="row-infor">
+        <p> Dân tộc: {(!!information && information.nation) || ''} </p>
+        <p> Nhập ngũ: {(!!information && format(information?.dateOfEnlistment, 'MM/yyyy')) || ''} </p>
+      </div>
+      <p> Đơn vị: {(!!information && information.unit) || ''} </p>
+      <div className="row-infor">
+        <p> Cấp bậc: {(!!information && information.rank) || ''} </p>
+        <p> Chức vụ: {(!!information && information.position) || ''} </p>
+      </div>
```

Comment:
!!information && <Wrapper>
hoặc
get lodash =))

Reply:
thanks for comments :3